project/categories/factory.py

```diff
@@ -1,19 +1,9 @@
 import factory
 from . import models
+from shared_factory.shared_factory import FreezeTimeModelFactory
 
-class UserFactory(factory.Factory):
+class CategoryFactory(FreezeTimeModelFactory):
     class Meta:
-        model = models.User
+        model = models.Category
 
-    first_name = 'John'
-    last_name = 'Doe'
-    admin = False
-
-# Another, different, factory for the same object
-class AdminFactory(factory.Factory):
-    class Meta:
-        model = models.User
-
-    first_name = 'Admin'
-    last_name = 'User'
-    admin = True
\ No newline at end of file
+    name = factory.Iterator(["Politics","Voting","Referendum"])
```

Comment:
Add new line at end of file
```suggestion
    name = factory.Iterator(["Politics","Voting","Referendum"])
    
```